tests/acceptance/29_simulate_mode/diff_mode.cf.expected

```diff
@@ -0,0 +1,124 @@
+ warning: All changes in files will be made in the 'WORKDIR/state/PID.changes' chroot
+===========================================================================
+'WORKDIR/tmp/rename-newname' is the new name of 'WORKDIR/tmp/rename-me'
+===========================================================================
+'WORKDIR/tmp/source-file' is a regular file
+Size: SIZE
+Access: (0600/rw-------)  Uid: (0/root)   Gid: (0/root)
+Access: TIMESTAMP
+Modify: TIMESTAMP
+Change: TIMESTAMP
+
+Contents of the file:
+These are my source file contents.
+\no newline at the end of file
+===========================================================================
+'WORKDIR/tmp/create-true' is a regular file
+Size: SIZE
+Access: (0600/rw-------)  Uid: (0/root)   Gid: (0/root)
+Access: TIMESTAMP
+Modify: TIMESTAMP
+Change: TIMESTAMP
+
+Contents of the file:
+===========================================================================
+'WORKDIR/tmp/insert-lines' is a regular file
+Size: SIZE
+Access: (0600/rw-------)  Uid: (0/root)   Gid: (0/root)
+Access: TIMESTAMP
+Modify: TIMESTAMP
+Change: TIMESTAMP
+
+Contents of the file:
+foobar
+===========================================================================
+'WORKDIR/tmp/SUBDIR' is a directory
+Size: SIZE
+Access: (0755/rwxr-xr-x)  Uid: (0/root)   Gid: (0/root)
+Access: TIMESTAMP
+Modify: TIMESTAMP
+Change: TIMESTAMP
+
+Directory contents:
+===========================================================================
+'WORKDIR/tmp/copy-from' is a regular file
+Size: SIZE
+Access: (0600/rw-------)  Uid: (0/root)   Gid: (0/root)
+Access: TIMESTAMP
+Modify: TIMESTAMP
+Change: TIMESTAMP
+
+Contents of the file:
+These are my source file contents.
+\no newline at the end of file
+===========================================================================
+'WORKDIR/tmp/delete-me' no longer exists
+===========================================================================
+'WORKDIR/tmp/sub-dir/./sub-file' no longer exists
+===========================================================================
+Only in WORKDIR/tmp/sub-dir/.: sub-file
```

Comment:
Where does this come from? I don't recall anything like this being in the code.

Reply:
Will check.

Reply:
Isn't it part of the `diff` output? Since `promises.cf.sub` deletes all files recursively under `sub-dir`?

```
    "$(G.testroot)$(const.dirsep)sub-dir/."
      delete => tidy,
      file_select => all,
      depth_search => recurse("inf");
```

Reply:
Oh, could be! I've never seen it, but then again I'm not using *diff* on folders that often.